src/node/rpc/serialization.h

```diff
@@ -9,13 +9,16 @@
 namespace ccf
 {
   DECLARE_JSON_ENUM(
-    GetSignedIndex::State,
-    {{GetSignedIndex::State::ReadingPublicLedger, "readingPublicLedger"},
-     {GetSignedIndex::State::ReadingPrivateLedger, "readingPrivateLedger"},
-     {GetSignedIndex::State::PartOfNetwork, "partOfNetwork"},
-     {GetSignedIndex::State::PartOfPublicNetwork, "partOfPublicNetwork"}})
-  DECLARE_JSON_TYPE(GetSignedIndex::Out)
-  DECLARE_JSON_REQUIRED_FIELDS(GetSignedIndex::Out, state, signed_index)
+    ccf::State,
+    {{ccf::State::uninitialized, "uninitialized"},
+     {ccf::State::initialized, "initialized"},
+     {ccf::State::pending, "pending"},
+     {ccf::State::partOfPublicNetwork, "partOfPublicNetwork"},
+     {ccf::State::partOfNetwork, "partOfNetwork"},
+     {ccf::State::readingPublicLedger, "readingPublicLedger"},
+     {ccf::State::readingPrivateLedger, "readingPrivateLedger"}})
+  DECLARE_JSON_TYPE(GetState::Out)
+  DECLARE_JSON_REQUIRED_FIELDS(GetState::Out, state, last_signed_index)
```

Comment:
Other recovery seqnos should also be serialised here?

Reply:
Doh, yes.